share/bash/checkfailures.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+quiet=$1
+status=$(bin/console messenger:failed:show 2>&1)
+errormsg=$(echo $status | grep 'There are 0 messages pending in the failure transport.')
+
+if [ "$quiet" == "quiet" ]; then
+    if [ -z "$errormsg" ]; then
+        echo $status | /bin/mailx -s "Failed message transport status" michael.williamson@tamucc.edu praneeth.pondicherry@tamucc.edu michael.vandeneijnden@tamucc.edu william.nichols@tamucc.edu rosalie.rossi@tamucc.edu
```

Comment:
is it okay to hardcode our emails to the git repo?

Reply:
Yes, don't like it!

Reply:
Also, can the subject NOT contain the word Failed, when I see it I think something failed.